omod/src/main/java/org/openmrs/module/fhir2/providers/ConditionFhirResourceProvider.java

```diff
@@ -70,4 +80,21 @@ public Condition getConditionByUuid(@IdParam @NotNull IdType id) {
 	public MethodOutcome createCondition(@ResourceParam Condition newCondition) {
 		return FhirServerUtils.buildCreate(conditionService.saveCondition(newCondition));
 	}
+	
+	@Search
+	//@SuppressWarnings("unused")
```

Comment:
Just delete this instead of commenting it out.

Reply:
I think that's an artifact of the fact that we've been writing code backwards, i.e., tests after the fact, so that the methods look like they're unused. Unfortunately, I'm as guilty of that as the next person.